client/my-sites/sidebar/sidebar.jsx

```diff
@@ -146,11 +146,21 @@ export class MySitesSidebar extends Component {
 			return null;
 		}
 
+		let sparkline;
+		if ( ! site.jetpack && ! this.isItemLinkSelected( 'stats' ) ) {
+			sparkline = (
+				<img
+					className="sidebar__sparkline"
+					src={ `https://${ site.wpcom_url }/wp-includes/charts/admin-bar-hours-scale-2x.php?masterbar=1` } />
```

Comment:
Or maybe this could be a selector returning false for Jetpack websites, that way we could just check for a truthy value.